crates/autopilot/Cargo.toml

```diff
@@ -16,9 +16,11 @@ path = "src/main.rs"
 anyhow = "1.0"
 async-trait = "0.1"
 clap = { version = "3.1", features = ["derive", "env"] }
+database = { path = "../database" }
 prometheus = "0.13"
 prometheus-metric-storage = { git = "https://github.com/cowprotocol/prometheus-metric-storage" , tag = "v0.4.0" }
 shared= { path = "../shared" }
+sqlx = { version = "0.6", default-features = false, features = ["bigdecimal", "chrono", "macros", "runtime-tokio-native-tls", "postgres"] }
```

Comment:
Do you think it makes sense to move all database things to `database` (including the `sqlx` dependency) and just having `orderbook` and `autopilot` call into it?

Reply:
The problem is that all things database currently depends on shared and I don't want the database crate to depend on shared. Even if we moved things out of shared this would still make the database crate depend on for example ethcontract for the Event types which doesn't feel right. My plan is to keep the database methods in the crate where they are used. orderbook will for example have the quote related methods and autopilot will have the event insertion.

Reply:
> Another idea I had was to have all everything in the database crate but only use types that sqlx natively supports.

This is how I originally imagined it TBH.